### ISSUE TITLE
Training ended with code 139 at 185215 step

### ISSUE THREAD

**Issue by ican24** [kan-bayashi/ParallelWaveGAN#157]
Hi @kan-bayashi ,

I am trying  ParallelWaveGAN trainng for test dataset, which generated Tacotron2 model in espnet TTS.
The training stopped with error code 139 in the middle of processing.
The voice generation with checkpoint-185000steps.pkl (and elder) is poor. Some words are skipped.
How may I investigate this case and fix it?
![Screenshot from 2020-05-27 12-45-48](https://user-images.githubusercontent.com/19214100/83020521-1cbc8e80-a018-11ea-81e8-284e26f12978.png)
![Screenshot from 2020-05-27 12-46-06](https://user-images.githubusercontent.com/19214100/83020551-2514c980-a018-11ea-87ae-8a8d604bb62e.png)

Thank you in advance

**Comment by kan-bayashi** [comment]
> I am trying ParallelWaveGAN trainng for test dataset, which generated Tacotron2 model in espnet TTS.

This means that you try to train the PWG with mel-spectrogram generated by Tacotron 2 using teacher forcing?

> The training stopped with error code 139 in the middle of processing.

Please attach the `train.log`.

**Comment by ican24** [comment]
> This means that you try to train the PWG with mel-spectrogram generated by Tacotron 2 using teacher forcing?

I used "How to make the recipe for your own dateset" from https://github.com/kan-bayashi/ParallelWaveGAN/blob/master/egs/README.md

train.log:
[train.log.zip](https://github.com/kan-bayashi/ParallelWaveGAN/files/4688929/train.log.zip)

**Comment by kan-bayashi** [comment]
Sorry, I misunderstood your post.
You made the recipe using your dataset, which is also used for the training of Tacotron 2, right?
I've never met this error code and no information in the log.
Actually, I have no idea.
Why don't you resume the training from the checkpoint?

> The voice generation with checkpoint-185000steps.pkl (and elder) is poor. Some words are skipped.

You decode with the tacotron2 generated features, did not perform analysis-synthesis, right? If so, it is better to separate the problem.
Basically, the word skipping does not happen in the vocoder.
That is a problem of the text2mel model.

**Comment by kan-bayashi** [comment]
One of the possible reason is the mismatch of cuda version.
What is the your cuda version and pytorch version?
And how did you install pytorch?

**Comment by ican24** [comment]
I found this info in my dmesg
`[307479.794024] parallel-wavega[5711]: segfault at 7efe0228e68d ip 00007efe8a28e68d sp 00007ffc1f731ae0 error 14 in libtorch.so[7efe88a16000+2f7bf000]
[307479.794028] Code: 48 fe ff ff 41 80 3e 00 c6 85 70 fe ff ff 00 c6 85 71 fe ff ff 00 74 12 41 0f b6 46 01 c6 85 70 fe ff ff 01 88 85 71 fe ff ff <0f> b6 8d ec fd ff ff 44 0f b6 85 e8 fd ff ff 48 8d 95 40 fe ff ff
`
I suspect that there is a hardware issue in my machine.
Let me to reboot and repeat training.

**Comment by kan-bayashi** [comment]
OK. If you find the reproducible procedure, please re-open.

**Comment by ican24** [comment]
Yes it was a hardware issue. After reboot It resumes to train normal way from the last checkpoint.
Sorry for bothering
The machine is a new. I did not meet any hardware trouble before it.
